pyproject.toml

```diff
@@ -4,17 +4,20 @@ version = "0.0.1"
 description = "Diabetes prediction"
 readme = "README.md"
 requires-python = ">=3.11, <3.12"
-dependencies = ["lightgbm>=4.5.0, <5",
-                "scikit-learn>=1.5.1, <2",
-                "cloudpickle>=3.0.0, <4",
-                "mlflow>=2.16.0, <3",
-                "numpy>=1.26.4, <2",
-                "pandas>=2.2.2, <3",
-                "pyarrow>=15.0.2, <16",
-                "cffi>=1.17.1, <2",
-                "scipy>=1.14.1, <2",
-                "matplotlib>=3.9.2, <4",
-                "databricks-feature-engineering>=0.6, <1"]
+dependencies = [
```

Comment:
for serving purpose, it is a good idea to pin packages to certain versions - check out versions we provide in week 2 branch.
This would break if you are going to deploy model serving with feature engineering lookup (it depends on pyarrow version 14)